recipes-extended/jailhouse/jailhouse_0.10.bb

```diff
@@ -55,23 +62,28 @@ USER_SPACE_CFLAGS = '${CFLAGS} -DLIBEXECDIR=\\\"${libexecdir}\\\" \
 TOOLS_SRC_DIR = "${S}/tools"
 TOOLS_OBJ_DIR = "${S}/tools"
 
+B = "${S}"
+
 do_compile() {
+
+   ln -sf ${PYTHON} ${STAGING_BINDIR_NATIVE}/python
```

Comment:
Why? We should get the using the right python without linking this.

Reply:
> Why? We should get the using the right python without linking this.

Why, is in commit message.
So you suggest to change into the scripts rather than use a link?

Reply:
Yes, because linking is easy to get wrong.